# Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.

# A type erased Protobuf message. Equivalent to the google.protobuf.Any message.

string  type_url    # Protobuf message type URL.
uint8[] value       # Packed Protobuf message instance.
